phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -629,6 +542,11 @@ public void start(CoprocessorEnvironment env) throws IOException {
         Tracing.addTraceMetricsSource();
         Metrics.ensureConfigured();
         metricsSource = MetricsMetadataSourceFactory.getMetadataMetricsSource();
+
+        //should I load it here or everytime in get TableFromCells.
```

Comment:
Getting a Table is a light weight operation. See conn.getTable() help
"This is a lightweight operation, pooling or caching of the returned Table is neither required nor desired."